Electron failed to install correctly, please delete node_modules/electron and try installing again (pnpm 10)

I am trying to build the latest `main` (commit 5297ee7da1e48b62bd3dfa48b4b0de514c6b67d5) of deltachat-desktop.

I changed `pnpm` version because I already have pnpm 10.0.0 installed:
```
 git diff
diff --git a/package.json b/package.json
index aa365a54..ee557070 100644
--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
   "engineStrict": true,
   "engines": {
     "node": "^20",
-    "pnpm": "^9.6.0"
+    "pnpm": "^10.0.0"
   },
   "pnpm": {
     "supportedArchitectures": {
```

I then try to build and run:
```
$ pnpm -w build:electron

> deltachat-desktop@1.50.1 build:electron /home/user/src/deltachat/deltachat-desktop
> pnpm --filter=@deltachat-desktop/target-electron build


> @deltachat-desktop/target-electron@1.50.1 build /home/user/src/deltachat/deltachat-desktop/packages/target-electron
> pnpm build:locales && pnpm build:backend && pnpm --filter=@deltachat-desktop/frontend build && pnpm build:compose-frontend


> @deltachat-desktop/target-electron@1.50.1 build:locales /home/user/src/deltachat/deltachat-desktop/packages/target-electron
> pnpm -w translations:convert


> deltachat-desktop@1.50.1 translations:convert /home/user/src/deltachat/deltachat-desktop
> node ./bin/build-shared-convert-translations.mjs ./_locales

+ Converting translations from xml to json...
+ converted 60 files in 374 ms


> @deltachat-desktop/target-electron@1.50.1 build:backend /home/user/src/deltachat/deltachat-desktop/packages/target-electron
> node ./bin/build.js

refs/heads/main main
{"VERSION":"1.50.1","BUILD_TIMESTAMP":1736955016495,"GIT_REF":"v1.33.0-1293-g5297ee7d"}

> @deltachat-desktop/frontend@ build /home/user/src/deltachat/deltachat-desktop/packages/frontend
> pnpm build:ts && pnpm build:themes && pnpm build:static && pnpm build:source-map-support && pnpm build:scss


> @deltachat-desktop/frontend@ build:ts /home/user/src/deltachat/deltachat-desktop/packages/frontend
> node ./bin/build-frontend-ts.mjs


> @deltachat-desktop/frontend@ build:themes /home/user/src/deltachat/deltachat-desktop/packages/frontend
> sass themes/light.scss static/fallback-theme.css --style=compressed --embed-source-map && node ./bin/build-frontend-themes.js


> @deltachat-desktop/frontend@ build:static /home/user/src/deltachat/deltachat-desktop/packages/frontend
> node ../../bin/copy.js ../../static ./html-dist && node ../../bin/copy.js ./static ./html-dist && node ../../bin/copy.js ../../images ./html-dist/images

+ copied all source files to "./html-dist"
+ copied all source files to "./html-dist"
+ copied all source files to "./html-dist/images"

> @deltachat-desktop/frontend@ build:source-map-support /home/user/src/deltachat/deltachat-desktop/packages/frontend
> node ./bin/build-frontend-source_map_support.js


> @deltachat-desktop/frontend@ build:scss /home/user/src/deltachat/deltachat-desktop/packages/frontend
> sass ./scss/manifest.scss ./html-dist/main.css --embed-source-map --style=compressed


> @deltachat-desktop/target-electron@1.50.1 build:compose-frontend /home/user/src/deltachat/deltachat-desktop/packages/target-electron
> node ../../bin/copy.js ../frontend/html-dist ./html-dist && node ../../bin/copy.js ./static ./html-dist && pnpm build:runtime-impl

+ copied all source files to "./html-dist"
+ copied all source files to "./html-dist"

> @deltachat-desktop/target-electron@1.50.1 build:runtime-impl /home/user/src/deltachat/deltachat-desktop/packages/target-electron
> pnpm esbuild --bundle --minify --keep-names --sourcemap --outdir=./html-dist runtime-electron/runtime.ts


  html-dist/runtime.js       33.4kb
  html-dist/runtime.js.map  121.0kb

⚡ Done in 9ms
$ pnpm -w start:electron

> deltachat-desktop@1.50.1 start:electron /home/user/src/deltachat/deltachat-desktop
> pnpm --filter=@deltachat-desktop/target-electron start


> @deltachat-desktop/target-electron@1.50.1 start /home/user/src/deltachat/deltachat-desktop/packages/target-electron
> NODE_OPTIONS=--enable-source-maps electron . --devmode --disable-http-cache --translation-watch

/home/user/src/deltachat/deltachat-desktop/node_modules/.pnpm/electron@32.1.0/node_modules/electron/index.js:17
    throw new Error('Electron failed to install correctly, please delete node_modules/electron and try installing again');
    ^

Error: Electron failed to install correctly, please delete node_modules/electron and try installing again
    at getElectronPath (/home/user/src/deltachat/deltachat-desktop/node_modules/.pnpm/electron@32.1.0/node_modules/electron/index.js:17:11)
    at Object.<anonymous> (/home/user/src/deltachat/deltachat-desktop/node_modules/.pnpm/electron@32.1.0/node_modules/electron/index.js:21:18)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/home/user/src/deltachat/deltachat-desktop/node_modules/.pnpm/electron@32.1.0/node_modules/electron/cli.js:3:18)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)

Node.js v20.11.1
/home/user/src/deltachat/deltachat-desktop/packages/target-electron:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @deltachat-desktop/target-electron@1.50.1 start: `NODE_OPTIONS=--enable-source-maps electron . --devmode --disable-http-cache --translation-watch`
Exit status 1
 ELIFECYCLE  Command failed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -50,7 +50,10 @@
         "ia32",
         "arm64"
       ]
-    }
+    },
+    "onlyBuiltDependencies": [
+      "electron"
+    ]
   },
   "devDependencies": {
     "@types/ignore-walk": "^4.0.3",

